mergebot_backend/db_publisher.py

```diff
@@ -0,0 +1,131 @@
+from datetime import datetime
+
+from mergebot_frontend import db
+from mergebot_frontend.models import Poller, QueuedItem, WorkItemStatus
+
+
+def publish_poller_status(project_name, status):
+    """publish_poller_status updates the status for a particular poller.
+    
+    Args:
+        project_name: Name of the project this poller belongs to.
+        status: Status to publish for this poller.
+    """
+    poller = Poller.query.filter_by(project_name=project_name).first()
+    if status == 'STARTED':
+        if not poller:
+            poller = Poller(project_name, datetime.now())
+            db.session.add(poller)
+        poller.status = 'STARTED'
```

Comment:
Think this was just an artifact of refactoring -- we didn't used to check if there was already a row for the poller.